project/src/components/comment-form/comment-form.tsx

```diff
@@ -0,0 +1,86 @@
+import { FormEvent, useState, ChangeEvent } from 'react';
+
+function CommentForm(): JSX.Element {
+  const [rating, setRating] = useState('0');
+  const [comment, setComment] = useState('');
+
+  return (
+    <form className="reviews__form form" action="#" method="post"
+      onSubmit={(evt: FormEvent<HTMLFormElement>) => {
+        evt.preventDefault();
+      }}
```

Comment:
Обработчики выноси выше не надо в шаблоне писать если это не однострочник
```suggestion
      onSubmit={handleSubmit}
```